common/static/common/js/components/views/paginated_view.js

```diff
@@ -0,0 +1,43 @@
+;(function(define) {
```

Comment:
In the past we haven't put the ``_view`` suffix on files that live under the ``views`` directory. However, I think this naming is actually clearer, so I'm fine with it.